content/docs/command-reference/exp/run.md

```diff
@@ -166,19 +166,16 @@ experiment we just ran (`exp-44136`).
 
 ## Example: Modify parameters on-the-fly
 
-You could modify a params file just like any other <abbr>dependency</abbr> and
-run an experiment on that basis. Since this is a common need, `dvc exp run`
-comes with the `--set-param` (`-S`) option built-in to update existing
-parameters. This saves you the need to manually edit the params file.
+`dvc exp run--set-param` (`-S`) saves you the need to manually edit the params
+file before running an experiment.
 
 ```dvc
 $ dvc exp run -S prepare.split=0.25 -S featurize.max_features=2000
 ...
 Reproduced experiment(s): exp-18bf6
-Experiment results have been applied to your workspace.
 ```
 
-To see the results, we can use `dvc exp diff` which compares both params and
+To see the results, we can use `dvc exp diff`, which compares both params and
```

Comment:
I think the sentence could be split into two. 

We can use `dvc exp diff` to see the result. It compares ...

Reply:
Sure. Also it should use 2nd person now that I see it:
```suggestion
To see the results, you can use `dvc exp diff`. It compares both params and
```